### ISSUE TITLE
[Installation] `pakohan/elasticmq` Error During Installation

### ISSUE THREAD

**Issue by uzair-ali10** [Cloud-CV/EvalAI#3629]
If you'd like to report a bug, please include the following information:
- expected behaviour
Successful Installation
- observed behaviour
Error
```❯ sudo docker-compose up --build
Pulling sqs (pakohan/elasticmq:)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]n
ERROR: pull access denied for pakohan/elasticmq, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

- steps to reproduce the problem
`git clone https://github.com/Cloud-CV/EvalAI.git evalai && cd "$_"`
`docker-compose up --build`
- which operating system and/or browser you're using, if relevant
Elementry OS (Based on Ubuntu 18.04)
- (if possible) copies of the console/server logs
![Screenshot from 2021-11-23 12-56-10](https://user-images.githubusercontent.com/72073401/142984878-134df7aa-66c3-4220-954b-7373cb355618.png)

PS: I'm already logged into Docker via terminal

It seems that [pakohan/elasticmq](https://hub.docker.com/r/pakohan/elasticmq) image is removed from docker hub and is the root cause of this problem
https://hub.docker.com/r/pakohan/elasticmq

**Comment by Manoj0Marmat** [comment]
@uzair-ali10 hello sir can i help you to solve this ?

**Comment by Manoj0Marmat** [comment]
@uzair-ali10 assign this to me

**Comment by Manoj0Marmat** [comment]
@uzair-ali10 fixed check

**Comment by Manoj0Marmat** [comment]
@uzair-ali10 sir can u please accept my pull request i wanna contribute

**Comment by Ram81** [comment]
Closing this issue. Fixed in PR #3638
